js/src/personal/maliha/Social.tsx

```diff
@@ -8,49 +9,52 @@ import styles from './Maliha.module.css'
 const iconMap = [
   {
     icon: FaInstagram,
-    size: '4.5em',
+    size: '3.5em',
     href: 'https://www.instagram.com/birb.erry/',
   },
   {
     icon: FaGithub,
-    size: '4.75em',
+    size: '3.5em',
     href: 'https://github.com/MalihaT111',
   },
   {
     icon: CiLinkedin,
-    size: '5em',
+    size: '4em',
     href: 'https://www.linkedin.com/in/maliha-tasnim-81864b178/',
   },
   {
     icon: CgMail,
-    size: '5em',
+    size: '4em',
     href: 'mailto:ayesiddika@gmail.com',
   },
 ]
 
 export function Social() {
   return (
-    <Stack
-      h={100}
-      align="left"
-      justify="center"
-      gap={75}
-      pos="absolute"
-      className={styles.icons}
-    >
-      {iconMap.map(({ icon: Icon, size, href }) => (
-        <SocialIcon
-          Icon={Icon}
-          size={size}
-          href={href}
-        />
-      ))}
-    </Stack>
+    <div>
+      <Stack
+        h={200}
+        align="left"
+        justify="center"
+        gap={55}
+        pos="fixed"
+        className={styles.icons}
+        pr={100}
+      >
+        {iconMap.map(({ icon: Icon, size, href }) => (
+          <SocialIcon
+            Icon={Icon}
+            size={size}
+            href={href}
+          />
+        ))}
+      </Stack>
+    </div>
   )
 }
 function SocialIcon({ Icon, size, href }: { Icon: any; size: any; href: any }) {
   const { hovered, ref } = useHover()
-  const iconColor = hovered ? '#a63b6a' : '#cb5c5c'
+  const iconColor = hovered ? '#c02366' : '#d93939'
```

Comment:
would the css :onhover selectors work here instead of tracking hover state from the UI? 
https://www.w3schools.com/cssref/sel_hover.php

Reply:
i think i run into more issues if i try using the onhover selectors, so i'd rather keep it like this for now